### PR TITLE
Add more details on update process

### PR DESCRIPTION
**Pull request by cedric-anne**
Ensure that this is clear that:
 - files extract must be done on an empty directory
 - `config` and `files` directory must be restored.

### CODE REVIEW

**Review by orthagh — APPROVED** [review]

**Comment by cedric-anne** [comment]
@cconard96 

Could you review this one ?

File: source/update.rst [hunk 1]
```diff
@@ -6,17 +6,26 @@ Update
...
+   * backup your `files` directory;
+   * backup your `config` directory, especially for your GLPI key file (`config/glpi.key` or `config/glpicrypt.key`) which is randomly generated.
 
-First, download latest GLPI version and extract files. GLPI update process is then automated. To start it, just go to your GLPI instance URI, or (recommended) use the :doc:`command line tools <command-line>`.
+Here are the steps to update GLPI:
 
-Once a new version will be installed; you will not be able to use the application until a migration has been done.
+* Download latest GLPI version.
+* Ensure the target directory is empty and extract files there.
+* Restore the previously backuped `config` and `files` directory.
```
Comment (cconard96) [inline source/update.rst:16]:
```suggestion
* Restore the previously backed up `config` and `files` directory.
```

**Review by cconard96 — CHANGES_REQUESTED** [review]

File: source/update.rst [hunk 2]
```diff
@@ -6,17 +6,26 @@ Update
    As for every update process, you have to backup some data before processing any upgrade:
 
    * **backup your database**;
-   * backup your files directory;
-   * backup your configuration (especially your GLPI key file `config/glpi.key` or `config/glpicrypt.key` if present).
+   * backup your `files` directory;
+   * backup your `config` directory, especially for your GLPI key file (`config/glpi.key` or `config/glpicrypt.key`) which is randomly generated.
```
Comment (trasher) [inline source/update.rst:10]: Maybe add a line about plugins and/or marketplace directories as well?

**Review by cconard96 — APPROVED** [review]

File: source/update.rst (hunk 2 above)
Comment (cedric-anne) [reply source/update.rst]: done

**Review by trasher — APPROVED** [review]

**Pull request merged**